Dockerfile.dapper

```diff
@@ -1,4 +1,4 @@
-FROM registry.suse.com/bci/bci-base:15.4
+FROM registry.suse.com/bci/bci-base:15.6
```

Comment:
to clarify: which version are we using, 15.6 or 15.5?

Reply:
for CI images we want to go already with 15.6, for production 15.5 and micro flavour
it is significant change which should eliminate any future potential security problems